packages/website/src/docs/getting-started.mdx

```diff
@@ -3,57 +3,36 @@ title: Getting Started
 pinned: 'true'
 ---
 
-<Divider size={2} my={6} />
-
 Radix is a Component Library built on top of [Radix System](/docs/radix-system).
 
-<Divider size={2} my={6} />
-
-### Install it
+### Installing Radix
 
-Radix is currently under heavy development, but if you'd like to use it anyway:
+Radix is currently under heavy development, but can install it by running either of these commands:
 
 ```js
-# npm
 npm install @modulz/radix
 ```
 
+or
```

Comment:
![image](https://user-images.githubusercontent.com/372831/60388448-96b36300-9aa9-11e9-93fa-f116b3f7935c.png)

Need a bit of spacing above the `or`

Reply:
That's fine for now. I want to optimise for the making the component pages gorgeous first. Then I'll loop back around and fix things in a more systematic way.